webcomponents/tool/src/main/frontend/packages/sakai-editor/LICENSE

```diff
@@ -0,0 +1,12 @@
+Copyright 2022 The Apereo Foundation Licensed under the
```

Comment:
Should this be 2024?

Reply:
I'm not actually sure. The code was not written this year.